core_backend/startup.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 python -m alembic upgrade head
-exec gunicorn -k uvicorn.workers.UvicornWorker -w 4 -b 0.0.0.0:8000 --preload main:app
+exec gunicorn -k main.Worker -w 4 -b 0.0.0.0:8000 --preload main:app
```

Comment:
Same. Change is to make sure Swagger UI works when using nginx (at `/api/docs`) and when using local dev setup (at `/docs`)